Assets/Prefabs/Resources/Storm MG/Storm Game.prefab

```diff
@@ -284,80 +284,6 @@ MonoBehaviour:
   m_EditorClassIdentifier: 
```

Comment:
it's strange that the Storm Game prefab is changing. did you disable it for testing maybe? We may need to revert this file back before merging

Reply:
oh right I changed its health bar for Kyle at the start of the semester (I forgot about that) she approved of it